pkg/unstruct/funcs.go

```diff
@@ -61,3 +62,27 @@ func patchUsingApplyStrategy(ctx context.Context, cli client.Client, source, tar
 func IsMarkedForDeletion(target *unstructured.Unstructured) bool {
 	return !target.GetDeletionTimestamp().IsZero()
 }
+
+func Patch(ctx context.Context, cli client.Client, target *unstructured.Unstructured) error {
```

Comment:
```suggestion
// Patch updates the specified Kubernetes resource by applying changes from the provided target object.
// In case of conflicts, it will retry using default strategy.
func Patch(ctx context.Context, cli client.Client, target *unstructured.Unstructured) error {
```